test/Interop/Cxx/library-evolution/allow-objc-in-cxx-mode-in-evolving-libraries.swift

```diff
@@ -1,6 +1,6 @@
 // RUN: %empty-directory(%t)
 // RUN: split-file %s %t
-// RUN: %target-swift-frontend %t/test.swift -I %t/Inputs -typecheck -enable-library-evolution -enable-experimental-cxx-interop -verify
+// RUN: %target-swift-frontend %t/test.swift -I %t/Inputs -typecheck -enable-library-evolution -enable-experimental-cxx-interop -disable-implicit-cxx-module-import -verify
```

Comment:
Why is the `disable-implicit-cxx-module-import` flag required in this test?

Reply:
```
<unknown>:0: error: unexpected warning produced: module 'Cxx' was not compiled with library evolution support; using it means binary compatibility for 'test' can't be guaranteed
```
Because we import `Cxx` module when enabling interop, enabling library evolution has to mean disabling this implicit import.

Reply:
And because I don't know how to handle error messages without source-location with `-verify` I thought I'd just disable it.

Reply:
I've refactored the decision to implicit import `Cxx` into a new `shouldImportCxx` method and made it default to `false` for when Library Evolution is enabled.

Reply:
Ah, I see! Thanks for explaining